client/src/App.js

```diff
@@ -30,11 +31,11 @@ class App extends Component {
   async generateBots() {
     const response = await fetchJson("/createbots/rating", {
       method: "POST",
-      body: JSON.stringify({ bots: this.state.bots }),
+      body: JSON.stringify({ bots: this.state.bots, timeStamp: this.state.timeStamp }),
```

Comment:
Just a thought, it might be easier for us to parse data later if we have 2 timestamps for the `start` and `end` of the round? That way, we won't need to join data from different rounds together later on to figure out the duration.

Something like 
````
{
    bots : this.state.bots,
    startTime: this.state.timeStamp,
    endTime: Date.now()
}
````